### PR TITLE
[Localization] BGM Changes in locales (+slight Spanish and French corrections)

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
Edited BGM names (+slight Spanish and French corrections)

## Why am I making these changes?
New OG musics just dropped and replaced some others, and so their titles too

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/ja/bgm-name.json [hunk 1]
```diff
@@ -128,7 +128,7 @@
   "tall_grass": "ポケダン空 のうむのもり",
   "temple": "ポケダン空 ばんにんのどうくつ",
   "town": "ポケダン空 ランダムダンジョン3",
-  "volcano": "ポケダン空 ねっすいのどうくつ",
+  "volcano": "Firel - Twisturn Volcano",
```
Comment (Chapybara-jp) [inline src/locales/ja/bgm-name.json:131]:
```suggestion
  "volcano": "Firel - Twisturn Volcano(曲がる折れる火山)",
```

File: src/locales/ja/bgm-name.json [hunk 2]
```diff
@@ -108,16 +108,16 @@
...
   "island": "ポケダン空 えんがんのいわば",
   "jungle": "Lmz - Jungle(ジャングル)",
   "laboratory": "Firel - Laboratory(ラボラトリー)",
   "lake": "ポケダン空 すいしょうのどうくつ",
   "meadow": "ポケダン空 そらのいただき（もり）",
   "metropolis": "Firel - Metropolis(大都市)",
   "mountain": "ポケダン空 ツノやま",
-  "plains": "ポケダン空 そらのいただき（そうげん）",
-  "power_plant": "ポケダン空 エレキへいげん",
+  "plains": "Firel - Route 888",
```
Comment (Chapybara-jp) [inline src/locales/ja/bgm-name.json:119]:
```suggestion
  "plains": "Firel - Route 888(888ばんどうろ)",
```

File: src/locales/ja/bgm-name.json [hunk 3]
```diff
@@ -108,16 +108,16 @@
...
+  "power_plant": "Firel - The Klink",
```
Comment (Chapybara-jp) [inline src/locales/ja/bgm-name.json:120]:
```suggestion
  "power_plant": "Firel - The Klink(ザ・ギアル)",
```

**Review by Chapybara-jp — APPROVED** [review]

**Review by flx-sta — DISMISSED** [review]

**Review by sodaMelon — APPROVED** [review]
I think it's better to keep the original name. (in Korean, no problem!)

**Comment by NicusPulcis** [comment]
i never planned to translate the OSTs names in italian but... why wasn't the placeholder even created..?

**Comment by Adri1** [comment]
> i never planned to translate the OSTs names in italian but... why wasn't the placeholder even created..?

They where but got removed. Everything still in English had to be deleted to properly use Gitlocalize (rip)

File: src/locales/ko/bgm-name.json [hunk 4]
```diff
@@ -107,17 +107,17 @@
   "forest": "불가사의 던전 하늘의 탐험대 검은 숲",
   "grass": "불가사의 던전 하늘의 탐험대 사과의 숲",
   "graveyard": "불가사의 던전 하늘의 탐험대 신비의 숲",
-  "ice_cave": "불가사의 던전 하늘의 탐험대 광대한 얼음산",
+  "ice_cave": "Firel - -50°C",
   "island": "불가사의 던전 하늘의 탐험대 연안의 암반",
   "jungle": "Lmz - Jungle",
   "laboratory": "Firel - Laboratory",
```
Comment (EnochG1) [inline src/locales/ko/bgm-name.json:113]:
```suggestion
  "jungle": "Lmz - 정글",
  "laboratory": "Firel - 연구소",
```

File: src/locales/ko/bgm-name.json [hunk 5]
```diff
@@ -107,17 +107,17 @@
...
-  "lake": "불가사의 던전 하늘의 탐험대 수정 동굴",
+  "lake": "Lmz - Lake",
```
Comment (EnochG1) [inline src/locales/ko/bgm-name.json:114]:
```suggestion
  "lake": "Lmz - 호수",
```

File: src/locales/ko/bgm-name.json [hunk 6]
```diff
@@ -107,17 +107,17 @@
...
   "meadow": "불가사의 던전 하늘의 탐험대 하늘 꼭대기 숲",
   "metropolis": "Firel - Metropolis",
   "mountain": "불가사의 던전 하늘의 탐험대 뿔산",
-  "plains": "불가사의 던전 하늘의 탐험대 하늘 꼭대기 초원",
-  "power_plant": "불가사의 던전 하늘의 탐험대 일렉트릭 평원",
-  "ruins": "불가사의 던전 하늘의 탐험대 봉인의 암반",
+  "plains": "Firel - Route 888",
+  "power_plant": "Firel - The Klink",
+  "ruins": "Lmz - Ancient Ruins",
```
Comment (EnochG1) [inline src/locales/ko/bgm-name.json:120]:
```suggestion
  "power_plant": "Firel - 기어르",
  "ruins": "Lmz - 고대 유적",
```

**Review by CodeTappert — DISMISSED** [review]

**Review by EnochG1 — APPROVED** [review]

**Review by CodeTappert — APPROVED** [review]

**Review by ben-lear — APPROVED** [review]

**Pull request merged**